www/layouts/course-lists/single.html

```diff
@@ -7,6 +7,7 @@
   <div class="container standard-width mx-auto mt-5">
     {{ partial "course_list" . }}
   </div>
+  {{ block "footer" . }} {{ partialCached "footer" . }} {{end}}
```

Comment:
```suggestion
  {{ block "footer" . }} 
      {{ partialCached "footer" . }} 
  {{end}}
```

This indentation can also be done on the other pages